src/transformers/trainer.py

```diff
@@ -421,7 +421,8 @@ def _remove_unused_columns(self, dataset: "datasets.Dataset", description: Optio
         signature = inspect.signature(self.model.forward)
         signature_columns = list(signature.parameters.keys())
         # Labels may be named label or label_ids, the default data collator handles that.
-        signature_columns += ["label", "label_ids"]
+        # And we need chinese reference inf when run wwm in chinese.
+        signature_columns += ["label", "label_ids", "chinese_ref"]
```

Comment:
No we can't have this here. Use `remove_unused_columns = True` to avoid the `Trainer` remove this column in your script but it should not change the code for every user of the library.

Reply:
Sorry I don't read these params carefully, now it has been fixed. 
And I wonder why we need those `LineByLineTextDataset ` if we have a `dataset` repo?

Reply:
> No we can't have this here. Use `remove_unused_columns = True` to avoid the `Trainer` remove this column in your script but it should not change the code for every user of the library.

`remove_unused_columns = False` ? :)